plugins/home/src/plugin.ts

```diff
@@ -210,3 +210,18 @@ export const HomePageRecentlyVisited = homePlugin.provide(
       import('./homePageComponents/VisitedByType/RecentlyVisited'),
   }),
 );
+
+/**
+ * A component to display specific Featured Docs.
+ *
+ * @public
+ */
+export const FeaturedDocs = homePlugin.provide(
+  createComponentExtension({
```

Comment:
Can you use the `createCardExtension` instead? Then it can be used in the Customizable Homepage where uses have the ability to add "widgets" and move them around as they like.

Reply:
Example of this: https://backstage.io/storybook/?path=/story/plugins-home-templates--customizable-template

Reply:
If you don't want to render the default card created by `createCardExtension`, you can also use the `createReactExtension` and it works too with the Customizable Homepage.

Reply:
thank you @awanlin & @drodil!
I went with  `createReactExtension` specifically because of types. 
I think there is a few changes that need to be made in `plugins/home-react/src/extensions.tsx` and some other files to avoid ts errors when allowing a `ReactNode` to be passed for the `title` as intended in [InfoCard](https://github.com/backstage/backstage/blob/3cca50a24060efc36164101a80c360c1fe4e3854/packages/core-components/src/layout/InfoCard/InfoCard.tsx#L133). 
We noticed this when we decided to use [StarredEntities](https://github.com/backstage/backstage/blob/3cca50a24060efc36164101a80c360c1fe4e3854/plugins/home/src/plugin.ts#L169) in our own Home Page. We plan on contributing this fix in the future if it is not already under way 👍